config/version

```diff
@@ -1 +1 @@
-v0.13.2-94-gf4c08be
+v0.13.3-14-ge94728c
```

Comment:
Actually, the current version should be displayed, like the one saved as a git tag, but it is always a commit version behind. I am thinking about doing the versioning manually. I don't need the commit hash anyway.